src/gui/owncloudgui.cpp

```diff
@@ -347,6 +348,15 @@ void ownCloudGui::addAccountContextMenu(AccountStatePtr accountState, QMenu *men
     actionOpenoC->setProperty(propertyAccountC, QVariant::fromValue(accountState->account()));
     QObject::connect(actionOpenoC, SIGNAL(triggered(bool)), SLOT(slotOpenOwnCloud()));
 
+    if (accountState->state() == AccountState::Connected) {
+        menu->addSeparator();
+
+        QString quotaInfo = tr("Fetching Quota");
+        auto actionQuotaInfo = menu->addAction(quotaInfo);
+        actionQuotaInfo->setEnabled(false);
+        new QuotaAction(actionQuotaInfo, accountState);
```

Comment:
@ckamm It already happens max every 30 seconds https://github.com/owncloud/client/blob/master/src/gui/quotainfo.h#L33-L36

We could of course have that job run in the background and have the info propegate to other parts of the code. Down side is that we will request more from the server.